dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
Is this interval used to request updates from the actual test server? Or does it request updates from the dashboard back-end? Might 1 second ever be a bit much?